src/app/test/helpers/html.ts

```diff
@@ -63,7 +63,7 @@ export function assertTooltip(target: HTMLElement, tooltip: string) {
   expect(attr).toBeTruthy();
   expect(attr.value.trim()).toBe(tooltip);
 
-  // TODO Add accessability expectations
+  // TODO Add accessability expectations (id, visible on highlight, etc.)
```

Comment:
Create a github issue for this, really should be done. Other assertions may also benefit from adding accessibility expectations.

Reply:
Issue Here: #395